.rubocop_todo.yml

```diff
@@ -1645,13 +1626,12 @@ Style/GuardClause:
     - 'lib/open_food_network/variant_and_line_item_naming.rb'
     - 'lib/spree/core/controller_helpers/order_decorator.rb'
     - 'lib/spree/core/controller_helpers/respond_with_decorator.rb'
-    - 'lib/tasks/karma.rake'
     - 'spec/features/admin/orders_spec.rb'
     - 'spec/support/delayed_job_helper.rb'
     - 'spec/support/request/distribution_helper.rb'
     - 'spec/support/request/shop_workflow.rb'
 
-# Offense count: 1255
+# Offense count: 1233
```

Comment:
Apparently there's 1233 offences for hash rockets, so it looks like `robocop_todo` is suggesting to disable the cop. The way we have it set up, I think the `rubocop_todo.yml` suggestions get applied first, then the `robocop.yml` rules override them. So that's why there's all the new offences.

On another note, seeing how this specific cop supports --auto-correct, can we just get rubocop to autocorrect those offences?

Reply:
The default is to disable a rule if there's more than 100 offences present, right?

Reply:
Something like that yes...